install/sample/gtp.yaml

```diff
@@ -3,11 +3,13 @@ kind: GlobalTrafficPolicy
 metadata:
   name: gtp-service1
   namespace: sample
+  annotations:
+    admiral.io/env: stage
   labels:
     identity: greeting
 spec:
   policy:
-    - dns: stage.greeting.global
+    - dns: default # default is a keyword, alternatively you can use `env` (ex: stage)
```

Comment:
Connecting to my previous comment ```Do you possible mean some other comparison here? Is it possible that the DNS = env?``` - could we maybe document this behavior a little bit more clearly? I find it a little counterintuitive that the DNS field should be the relevant environment (and it's distinctly different from how the docs say things work: https://github.com/istio-ecosystem/admiral/blob/master/docs/Architecture.md)